v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FConstants.scala

```diff
@@ -0,0 +1,244 @@
+package org.broadinstitute.monster.etl.v2f
+
+/**
+  * constants for v2f that allow for transforms of the converted tsv to json
```

Comment:
In general, scaladocs should be capitalized and end with `.` on their first line.